packages/cloud-functions/src/cico/transakWebhook.ts

```diff
@@ -0,0 +1,84 @@
+import * as functions from 'firebase-functions'
+import jwt from 'jsonwebtoken'
+import { trackEvent } from '../bigQuery'
+import { BIGQUERY_PROVIDER_STATUS_TABLE, TRANSAK_DATA } from '../config'
+import { saveTxHashProvider } from '../firebase'
+import { CashInStatus, Provider } from './Provider'
+
+enum TransakEvent {
+  Created = 'ORDER_CREATED',
+  Failed = 'ORDER_FAILED',
+  Completed = 'ORDER_COMPLETED',
+}
+
+function trackTransakEvent(body: any) {
+  const {
+    eventID,
+    webhookData: { id, walletAddress, statusReason },
+  } = body
+  if (eventID === TransakEvent.Created) {
+    trackEvent(BIGQUERY_PROVIDER_STATUS_TABLE, {
+      id,
+      provider: Provider.Transak,
+      status: CashInStatus.Started,
+      timestamp: Date.now() / 1000,
+      user_address: walletAddress,
```

Comment:
Nit: You can define the common properties outside of the `if` blocks to dry this up a bit